CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ This changelog tracks the releases of docker-credential-helpers.
 This project includes different binaries per platform.
 The platform released is identified after the tag name.
 
+## v0.6.1 (Go client, Mac OS X)
```

Comment:
could you move the changelog update and version bump to a separate PR?

I think there's more changes since the last release, and it's better to do it separate to review that properly

Reply:
OK.